layouts/partials/head.html

```diff
@@ -119,6 +119,10 @@
         </script>
     {{ end }}
 
+    <!-- Preload the component JavaScript to help our Google Page Speed score. -->
+    <link rel="preload" href="/js/components/components.esm.js" as="script" crossorigin>
```

Comment:
I'm fairly certain without the crossorigin the prefetch doesn't work because we need to fetch the script "anonymously" but I am going to commit this and confirm that.